gem/lib/pupperware/spec_helper.rb

```diff
@@ -235,6 +239,11 @@ def get_container_name(container)
     inspect_container(container, '{{.Name}}')
   end
 
+  def get_container_labels(container)
+    json = inspect_container(container, '{{json .Config.Labels}}')
```

Comment:
support null here too?

Reply:
ha looks like that's what the CI failure might be about

Reply:
Since this is always a collection, I don't think so... it just returns an empty collection.

But I think Docker may automatically add at least one label anyhow.